src/main/java/com/google/javascript/clutz/ImportRenameMapBuilder.java

```diff
@@ -129,7 +131,8 @@ private static String getGoogModuleId(Node astRoot) {
    * goog.module.get() assignments or destructuring assignments, parsing them, and generating the
    * mappings from local symbol names to exported symbol names.
```

Comment:
extend the comment with method local description of the purpose of knownGoogProvides